### PR TITLE
Update lsilveirajunior-stat6863.md

### PR DESCRIPTION
**Pull request by lsilveirajunior-stat6863**

### CODE REVIEW

File: week-01-reflections/lsilveirajunior-stat6863.md [hunk 1]
```diff
@@ -18,8 +29,24 @@
 ```
 
 
-[place your recipes exploration here, and delete this line]
+**Recipe: Class greeting;
+
+data _null_;
```
Comment (prof-lankham) [inline week-01-reflections/lsilveirajunior-stat6863.md]: These are not the versions of the recipes given in the course videos

File: week-01-reflections/lsilveirajunior-stat6863.md [hunk 2]
```diff
@@ -2,10 +2,21 @@
 # Questions about Problems and Recipes
 
 
-
-[place your questions/answers here, and delete this line]
-
-
+[Course Structure Quiz, Problem 5]
+∗ Question (<lsilveirajunior-stat6863 >): Will the final exam have multiple choice questions similar to the exercises done during the 10-week program or new material will be added to it?
```
Comment (prof-lankham) [inline week-01-reflections/lsilveirajunior-stat6863.md]: This is not the correct Q/A format: The asterisk you're using does not render as a bullet point in Markdown, and there should not be angle brackets around your GitHub username

File: week-01-reflections/lsilveirajunior-stat6863.md [hunk 3]
```diff
@@ -2,10 +2,21 @@
...
+∗ Question (<lsilveirajunior-stat6863 >): Will extra credits be provided for implemented suggestions and corrections or just the participation and interaction will allow the student to get better grades?
+  
+[Course Structure Quiz, Problem 9]
+∗ Question (<lsilveirajunior-stat6863 >): After using Slack, I feel that it's a much easier interface to communicate than through emails. DMing through Slack would be an appropriate tool in a workplace environment or emails are more recommended? 
+  
+[Course Structure Quiz, Problem 2]
+∗ Question (<lsilveirajunior-stat6863 >): As a question and also a suggestion, wouldn't it be easier to send the weekly forum post through Slack or GitHub as a way of concentrating our submissions in a single platform?
+  
+[Course Structure Quiz, Problem 6]
+∗ Question (<lsilveirajunior-stat6863 >): How much weight will Team-based Problem Solving Achievement affect the final grade? If the group doesn't perform well, is it possible to perform other tasks to achieve better grades?
```
Comment (prof-lankham) [inline week-01-reflections/lsilveirajunior-stat6863.md]: Questions also need to be added for recipes

**Comment by prof-lankham** [comment]
Also, PR title is incorrect

**Comment by prof-lankham** [comment]
PR stale; being closed

**Comment by lsilveirajunior-stat6863** [comment]
Hello Prof. lankham,

I've performed the changes already. What is wrong with the title?


Best,

Luiz.

**Pull request closed without merging**